### PR TITLE
Fix lost state arguments

### PR DESCRIPTION
**Pull request by ngoryachev**

### CODE REVIEW

**Comment by ngoryachev** [comment]
@isnifer bump!

File: src/Router.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ export default class TipsiRouter {
     const location = {
       pathname: compile(path)(paramsOrOptions),
       search: query && stringify(query, { addQueryPrefix: true }),
+      state: paramsOrOptions,
```
Comment (isnifer) [inline src/Router.js]: Before you change it was https://github.com/tipsi/tipsi-router/blob/aff5589412b4d9632a2ecd085377c923f585d299/src/Router.js#L67
To be compatible with `ex-navigation`

Comment (ngoryachev) [reply src/Router.js]: i thought what is it =) Returned back

**Review by isnifer — APPROVED** [review]
LGTM

**Pull request merged**